cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+                                 /* Delete this field when you are using destinationPolicy. */
+      "destinationType": "",     /* The pattern in use by nodes that should receive this object. */
+                                 /* If omitted (and if destinationsList is omitted too) the object is broadcast to all known nodes. */
+                                 /* Delete this field when you are using policy. */
+      "destinationsList": null,  /* The list of destinations as an array of pattern:nodeId pairs that should receive this object. */
+                                 /* If provided, destinationType and destinationID must be omitted. */
+                                 /* Delete this field when you are using policy. */
+      "destinationPolicy": {     /* The policy specification that should be used to distribute this object. */
+                                 /* Delete these fields if the target node is using a pattern. */
+        "properties": [          /* A list of policy properties that describe the object. */
+          {
+            "name": "",
+            "value": null,
+            "type": ""           /* Valid types are string, bool, int, float, list of string (comma separated), version. */
+                                 /* Type can be omitted if the type is discernable from the value, e.g. unquoted true is boolean. */
+          }
+        ],
+        "constraints": [         /* A list of constraint expressions of the form <property name> <operator> <property value>, separated by boolean operators AND (&&) or OR (||). */
+          ""
+        ],
+        "services": [            /* The service(s) that will use this object. */
+          {
+            "orgID": "",         /* The org of the service. */
+            "serviceName": "",   /* The name of the service. */
+            "arch": "",          /* Set to '*' to indcate services of any hardware architecture. */
+            "version": ""        /* A version range. */
+          }
+        ]
+      },
+      "expiration": "",          /* A timestamp/date indicating when the object expires (it is automatically deleted). The timestamp should be provided in RFC3339 format.  */
+      "version": "",             /* Arbitrary string value. The value is not semantically interpreted. The Model Management System does not keep multiple version of an object. */
+      "description": "",         /* An arbitrary description. */
+      "activationTime": ""       /* A timestamp/date as to when this object should automatically be activated. The timestamp should be provided in RFC3339 format. */
+    }
+ 
+Use `hzn mms object new >> my_metadata.json` to copy the template to a file named `my_metadata.json`.  (Or you can just copy the template from terminal and paste to a file). Then fill out the field in `my_metadata.json`, and save file.
+
+##### Send MMS to node running with policy
+
+My edge node `an12345` is using policy. `usehello2` is one of the services running on it. 
```

Comment:
This is good. I'm not sure we want to use usehello2 but whatever we use we have to test it to make sure it works. Checkout this example that already exists: https://github.com/open-horizon/examples/tree/master/edge/services/helloMMS maybe you can just use it, we know it works because Troy maintains it.